Formula/c/chocolate-doom.rb

```diff
@@ -6,6 +6,11 @@ class ChocolateDoom < Formula
   license "GPL-2.0-only"
   head "https://github.com/chocolate-doom/chocolate-doom.git", branch: "master"
 
+  livecheck do
+    url :stable
+    regex(/^chocolate-doom-(\d+(?:\.\d+)+)$/i)
```

Comment:
```suggestion
    regex(/^chocolate-doom[._-]v?(\d+(?:\.\d+)+)$/i)
```

Just some small tweaks to bring this in line with typical regex patterns.